### ISSUE TITLE
My vscode say can't activate the debugger for unity

### ISSUE THREAD

**Issue by panit100** [OmniSharp/omnisharp-vscode#3628]
My vscode say can't activate the debugger for unity extension because it depends on an unknown ''ms-vscode.csharp' extension but i already install c# Extensions. And my vscode show pop-up say "The 'c#' extension is recommended for this file type" but when i click install it can't find '@id:ms-vscode.csharp'.

**Comment by timopitt** [comment]
I have the exact same problem here since today. I tried to de- and reinstall vs code with no change.

**Comment by KkachiGames** [comment]
I was having the same issue. I found this solution [(#3623)](https://github.com/OmniSharp/omnisharp-vscode/issues/3623#issuecomment-595561225), and it worked for me. Good luck!

**Comment by MrLord9000** [comment]
For some reason Microsoft decided to change extension publisher name from ms-vscode to ms-dotnettools in version 1.21.13 and broke most of dependent extensions. I think downgrade to 1.21.12 is the way to go.

**Comment by ThomasMoonKang** [comment]
Just note you can't download an old version using the "install another version" button from the extensions panel. It just redirects to download 1.21.13 (presumably because of the name change). How infuriating. The solution @KkachiGames linked to works though!

**Comment by jossinger** [comment]
I was having the same issue and after reverting back to 1.21.12 both the error "c# extension is recommended for this file type" and the error "can't activate the debugger for unity" have disappeared, however, intellisense still doesn't seem to be working?

**Comment by ngtrhieu** [comment]
![Screenshot 2020-03-10 at 2 48 34 PM](https://user-images.githubusercontent.com/8876396/76286927-3e8e5b00-62de-11ea-9087-bb4ef4c5b714.png)
This is due to Unity Debugger looking for ms-vscode package where the new C# package name has been renamed to ms-dotnettools

**Comment by cartermp** [comment]
Thanks for the report. This issue is because the publisher name for the C# extension was changed, and this broke the unity extension. There is a PR out on the unity extension to update, and once that is merged, this shouldn't reproduce. Closing as a duplicate of https://github.com/OmniSharp/omnisharp-vscode/issues/3620

**Comment by Thaina** [comment]
I think this is seriously unthinkable and unexpectable that you just change something that may affect dependency chain of extension. What the reason for this? And should we do something about this issue in vscode?

**Comment by panit100** [comment]
> I was having the same issue. I found this solution [(#3623)](https://github.com/OmniSharp/omnisharp-vscode/issues/3623#issuecomment-595561225), and it worked for me. Good luck!

thank It work now
